### PR TITLE
Release v2024.2.0

### PR DESCRIPTION
**Pull request by r-n-o**
Routine update to sign the latest swagger spec. See commits, I had to first get back to a clean slate, then updated to the latest spec, then ran `make generate` again.

There's also an issue with our current nix flake that I'm going to address separately: this was generated with go-swagger 0.30.5: apparently 0.30.4 has issues!

### CODE REVIEW

File: pkg/api/models/create_wallet_intent.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ type CreateWalletIntent struct {
 	Accounts []*WalletAccountParams `json:"accounts"`
 
 	// Length of mnemonic to generate the Wallet seed. Defaults to 12. Accepted values: 12, 15, 18, 21, 24.
-	MnemonicLength string `json:"mnemonicLength,omitempty"`
+	MnemonicLength int32 `json:"mnemonicLength,omitempty"`
```
Comment (andrewkmin) [inline pkg/api/models/create_wallet_intent.go:28]: this should help unblock one of our clients!

File: api/public_api.swagger.json [hunk 2]
```diff
@@ -2713,8 +2713,8 @@
           "description": "A list of wallet Accounts."
         },
         "mnemonicLength": {
-          "type": "string",
-          "format": "uint64",
+          "type": "integer",
```
Comment (andrewkmin) [inline api/public_api.swagger.json:2716]: shoutout to our customer for notifying us of this :)

File: api/public_api.swagger.json [hunk 3]
```diff
@@ -5677,8 +5677,8 @@
       "tags": ["Organizations", "Invitations", "Policies", "Features"]
     },
     {
-      "name": "PRIVATE KEYS",
-      "tags": ["Wallets", "Signers", "Private Keys", "Private Key Tags"]
+      "name": "WALLETS AND PRIVATE KEYS",
+      "tags": ["Wallets", "Signing", "Private Keys", "Private Key Tags"]
```
Comment (andrewkmin) [inline api/public_api.swagger.json:5681]: will need to double check that linking/etc all works as expected on the docs page. it should! 🤞

**Review by andrewkmin — APPROVED** [review]
🙏

File: api/public_api.swagger.json (hunk 3 above)
Comment (r-n-o) [reply api/public_api.swagger.json:5681]: I had to search/replace all "Signers" and replace with "signing", we had an example using the signers namespace!

**Pull request merged**